bundles/org.openhab.binding.nuki/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -47,14 +70,17 @@
 			<channel id="lock" typeId="smartlockLock"/>
 			<channel id="lockState" typeId="smartlockState"/>
 			<channel id="lowBattery" typeId="system.low-battery"/>
+			<channel id="keypadLowBattery" typeId="system.low-battery"/>
+			<channel id="batteryLevel" typeId="system.battery-level"/>
+			<channel id="batteryCharging" typeId="smartLockBatteryCharging"/>
 			<channel id="doorsensorState" typeId="smartlockDoorState"/>
 		</channels>
+		<properties>
+			<property name="name">-</property>
+			<property name="firmwareVersion">-</property>
+		</properties>
+		<representation-property>nukiId</representation-property>
 		<config-description>
-			<parameter name="nukiId" type="text" required="true">
-				<label>Nuki ID</label>
-				<description>The 8-digit hexadecimal string that identifies the Nuki Smart Lock. Look it up on the sticker on the
-					back of the Nuki Smart Lock (remove mounting plate).</description>
-			</parameter>
```

Comment:
Is this removed by intention?

Reply:
Yes, in the new version of binding user is not supposed to edit this property. The only way to add new Smart lock or Opener is using automatic discovery, which will read nuki id from bridge API.

Reply:
The user should always be able to use textual configuration, also if the parameter can only be obtained via discovery. The user could get the parameter after discovery and convert his/her config into textual.

Reply:
That's a good point, I haven't thought about textual configuration. The only small issue is that all places user can read the nukiId (like the mobile app) show it as hexadecimal value, but the API always returns value in decimal (but somehow accepts both values) so it might be confusing in which format the nukiId is and whether you can compare it with value returned from API.

Reply:
If the API accepts both formats, maybe you can do, too? Otherwise, users using textual configuration should be able to convert a decimal number to hex and vice versa.

Reply:
Ok, I've put the configuration parameter back. If it is set as hexadecimal, it will be converted (for example by previous binding version)